src/sinks/datadog/metrics.rs

```diff
@@ -74,13 +76,27 @@ struct DatadogRequest<T> {
 }
 
 impl DatadogConfig {
-    fn get_endpoint(&self) -> &str {
+    fn get_endpoint(&self) -> String {
```

Comment:
This might be worth a comment to what it's doing. I was surprised to see `PKG_VERSION` going into the endpoint hostname.

Reply:
Indeed, it's worth mentioning. I will add a note before merging the PR.